ios/FluentUI/Utilities/ContentHeightResolutionContext.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+@objc(MSFContentHeightResolutionContext)
+public class ContentHeightResolutionContext: NSObject {
```

Comment:
bit unclear to me why this won't be a simple struct and it can't live within bottomsheet folder because i don't know what the usage would be outside of this context

Reply:
Why not a simple struct - I wish I could use that, but unfortunately those don't work with @objc.

Why not bottom sheet specific:
One reason is that none of the height resolver pattern is bottom sheet specific, it could be used in other classes. 

Another, more valid reason is that I want to use this internally for presentation configuration, decoupled from the actual presentation. This way app clients that want to present something can provide a height behavior to Silhouette without necessarily knowing about (or committing to) a bottom sheet controller being used behind the scenes to do the presentation.

Reply:
If you think it would be cleaner to have this under the bottom sheet folder, I'd be happy to do that.

Reply:
Wait, I think I could make this swift-only, which would let me keep it as a struct. That's probably a greater benefit than the above reusability stuff...

Reply:
Moved back under bottom sheet folder. Kept an objc compatible class after a suggestion from Mike.